lib/Widgets/AlertView.vala

```diff
@@ -73,12 +73,16 @@ public class Granite.Widgets.AlertView : Gtk.Grid {
     /**
      * The icon name
      */
-    public string icon_name {
+    public string? icon_name {
         owned get {
             return image.icon_name;
```

Comment:
It it valid to reference a property of a (possibly) destroyed widget?  Do we need to be able to set the icon name after construction of the AlertView?